src/main/app/check-your-answers/CheckYourAnswersPostController.ts

```diff
@@ -1,26 +1,26 @@
 import autobind from 'autobind-decorator';
-import { Response } from 'express';
+import { NextFunction, Response } from 'express';
 
 import { getNextStepUrl } from '../../steps';
 import { Case } from '../case/case';
-import { ApplicationType, CITIZEN_INVITE_APPLICANT_2 } from '../case/definition';
+import { ApplicationType, CITIZEN_INVITE_APPLICANT_2, CITIZEN_SUBMIT } from '../case/definition';
 import { AppRequest } from '../controller/AppRequest';
 import { AnyObject, PostController } from '../controller/PostController';
 import { Form } from '../form/Form';
 
 @autobind
-export class InviteApplicant2PostController extends PostController<AnyObject> {
+export class CheckYourAnswersPostController extends PostController<AnyObject> {
   constructor(protected readonly form: Form) {
     super(form);
   }
 
-  public async post(req: AppRequest<AnyObject>, res: Response): Promise<void> {
+  public async post(req: AppRequest<AnyObject>, res: Response, next?: NextFunction): Promise<void> {
```

Comment:
i think this all file can be simiplied similiar to how i do it here [link](https://github.com/hmcts/nfdiv-frontend/pull/697/files#diff-a725046819a0e0f28e62dd6ff56fd4ff7eea0abba5fcf80fb97220d93a120319R15)

Reply:
:+1: Nice, I've take your new implementation